Emma/emma_libs/memoryManager.py

```diff
@@ -51,21 +54,24 @@ def __init__(self, projectName, configurationPath, mapfilesPath, outputPath, ana
             self.removeUnmatched = removeUnmatched
             self.noPrompt = noPrompt
             self.noResolveOverlap = noResolveOverlap
+            self.graphics = graphics
 
-    def __init__(self, projectName, configurationPath, mapfilesPath, outputPath, analyseDebug, createCategories, removeUnmatched, noPrompt, noResolveOverlap):
+    def __init__(self, projectName, configurationPath, mapfilesPath, outputPath, analyseDebug, createCategories,
+                 removeUnmatched, noPrompt, noResolveOverlap, graphics):
         # pylint: disable=too-many-arguments
         # Rationale: We need to initialize the Settings, so the number of arguments are needed.
 
         # Processing the command line arguments and storing it into the settings member
-        self.settings = MemoryManager.Settings(projectName, configurationPath, mapfilesPath, outputPath, analyseDebug, createCategories, removeUnmatched, noPrompt, noResolveOverlap)
+        self.settings = MemoryManager.Settings(projectName, configurationPath, mapfilesPath, outputPath, analyseDebug,
+                                               createCategories, removeUnmatched, noPrompt, noResolveOverlap, graphics)
         # Check whether the configuration and the mapfiles folders exist
         Emma.shared_libs.emma_helper.checkIfFolderExists(self.settings.mapfilesPath)
-        self.configuration = None                   # The configuration is empty at this moment, it can be read in with another method
+        self.configuration = None  # The configuration is empty at this moment, it can be read in with another method
         # memoryContent [dict(list(memEntry))]
         # Each key of this dict represents a configID; dict values are lists of consumerCollections
         # consumerCollection: [list(memEntry)] lists of memEntry's; e.g. a Section_Summary which contains all memEnty objects per configID)
-        self.memoryContent = None                   # The memory content is empty at this moment, it can be loaded with another method
-        self.categorisation = None                  # The categorisation object does not exist yet, it can be created after reading in the configuration
+        self.memoryContent = None  # The memory content is empty at this moment, it can be loaded with another method
+        self.categorisation = None  # The categorisation object does not exist yet, it can be created after reading in the configuration
```

Comment:
Keep initial indentation.

```suggestion
        self.categorisation = None                  # The categorisation object does not exist yet, it can be created after reading in the configuration
```